code/modules/mob/living/emote.dm

```diff
@@ -604,6 +609,7 @@
 	message = "spews something softly."
 	emote_type = EMOTE_AUDIBLE
 	vary = TRUE
+	cooldown = (60 SECONDS)
 	mob_type_allowed_typecache = list(/mob/living/carbon/human, /mob/living/brain, /mob/living/silicon)
 
 /datum/emote/living/ffff/get_sound(mob/living/user)

```

Comment:
add a cooldown to ffff as well

Reply:
unless you, too, are a man of culture who likes the gloves of the north star *ffff battlecry meme

Reply:
That's... that's what that is, yes.

Reply:
wait, I didn't realize that you did, in fact, add a cooldown

Reply:
never mind, then